python/lsst/sims/GalSimInterface/galSimInterpreter.py

```diff
@@ -720,14 +732,23 @@ def __init__(self, obs_metadata=None, detectors=None, bandpassDict=None,
                                        treering_func=det.tree_rings.func,
                                        transpose=True)
 
-    def drawObject(self, gsObject):
+    def drawObject(self, gsObject, max_flux_simple=100, sensor_limit=200):
```

Comment:
I just realized that in order to maintain compatibility with code that uses the `GalSimInterpreter` class, i.e., without the silicon model and other calculations enabled, the `drawObject` function needs to have these keywords  in the function signature even though they would not be used.   We should obviously refactor these classes at some point, but just adding those keywords to that function would be expedient.

Reply:
Done.